### ISSUE TITLE
Problem with keyboard shortcut

### ISSUE THREAD

**Issue by meelten** [woodruffw/ff2mpv#96]
## Describe the bug

When I push CTRL+SHIFT+V on website with video nothing happens despite the ff2mpv toolbar button works fine. By sheer accident I discovered that when switching to Firefox's "responsive design mode" (CTRL+SHIFT+M) and then using shortcut makes it works. It also works when I close "responsive design mode" **as long as I won't click mouse anywhere on the site or even scroll it**. That by my layman understanding makes me think that shortcut works only when website is "out of focus", otherwise something interferes with shortcut but I have no clue how debug it further.

## Reproduction steps

1. Go to whatever URL with video working in ff2mpv in Firefox.
2. Push CTRL+SHIFT+V
3. If nothing happens push CTRL+SHIFT+M and then again CTRL+SHIFT+V without using mouse anywhere in-between.

## Platform information

 - Arch Linux (KDE Plasma Desktop)
 - Firefox 110
 - MPV 0.35.1
 - yt-dlp 2023.02.17

**Comment by woodruffw** [comment]
Hmm, that's interesting -- I don't ever use the shortcut, but I can confirm this behavior: if I go to a YouTube link and press `C-s-V`, nothing happens unless I first enter responsive design mode first.

You're right that it's probably being masked somewhere. 

My repro case:

* Ubuntu 22.04 (i3wm)
* Firefox 110
* MPV 0.34.1
* yt-dlp 2023.02.17

**Comment by woodruffw** [comment]
(Also, thank you for reporting, and for filling our the reproduction and platform information!)

**Comment by woodruffw** [comment]
Tried with a SoundCloud URL as well, so this doesn't seem to be isolated to a single website.

**Comment by meelten** [comment]
Yeah it happens on every website and since it's not just me I guess it must be Firefox that somehow blocks this shortcut. Actually I never used it previously either so can't tell when it stopped working to see what changed then.

Probably using different combination would fix this?

**Comment by DanSM-5** [comment]
I think I may found the reason of the shortcut issue on firefox. The combination `ctrl+shift+v` is the default for "paste as plain text" (sometimes refer to "paste without formatting").

That may take priority when the window is in focus.

ref: https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly

![image](https://github.com/user-attachments/assets/0ce39f7c-4f74-4a70-8940-0b4fcdfac562)
